# List if bounding boxes inferred from an image by an object detection
# algorithm

# Header set by the object detection algorithm
Header header

# Header copied from the image which generated this message
Header image_header

# List of bounding boxes inferred from the image which generated this message
BoundingBox[] bounding_boxes